### PR TITLE
[Checklist] Add an example for M1000

### PR DESCRIPTION
**Pull request by RobDolinMS**
Adds an example
Links to the original file
NOTE: Removes the "tags": ["Profiles"], object from the original JSON without "..." for readability in the checklist

Signed-off-by: Rob Dolin <robdolin@microsoft.com>

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes. - n/a, I'm starting to add examples to non-automated cases per conversation with ADX team
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec). - n/a

### Quality of Swagger - n/a

### CODE REVIEW

**Review by veronicagg — APPROVED** [review]

**Review by salameer — CHANGES_REQUESTED** [review]
Hi Rob,

This looks good, however the team is currently working at more rules and the ARM has recently added more rules. my concerns here is if we dont find a way to have this collapsible then the checklist would become humongous .

Also I think we need to show what's a "Bad example" to give the full picture :)

**Comment by RobDolinMS** [comment]
Thanks @salameer.  I'll look into how to do collapsing.

**Comment by salameer** [comment]
@kirthik  and @RobDolinMS 

I think we might wanna hold on on this until we have all the content available in the Checklist thoguhts?

**Comment by RobDolinMS** [comment]
>I [@salameer] think we might wanna hold on on this until we have all the content available in the Checklist thoguhts?

@salameer Would you please clarify what you mean by "all the content?"

**Comment by salameer** [comment]
HI @RobDolinMS 

I was  reffering to the rest of the rules that are currently not  documented or listed into the checklist

File: documentation/swagger-checklist.md [hunk 1]
```diff
@@ -84,6 +84,13 @@ The Azure Developer Experience team will continue to deliver rich tools to valid
 
 - [ ] **M1000**: Plural names in parameters and model properties MUST only be used to represent collections.
 
+  Example from [arm-cdn](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-cdn/2016-10-02/swagger/cdn.json):
+  ```json
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Cdn/profiles": {
```
Comment (kirthik) [inline documentation/swagger-checklist.md]: @RobDolinMS : Example here does not point to plural names. Am i confused here? Could you please point me to the parameters/model properties which are collections?

**Comment by RobDolinMS** [comment]
Closing this as I understand the checklist format is getting a big update soon.

**Pull request closed without merging**